chronograf.go

```diff
@@ -380,7 +380,7 @@ type Dashboard struct {
 
 // DashboardCell holds visual and query information for a cell
 type DashboardCell struct {
-	ID      string           `json:"-"`
+	ID      string           `json:"i"`
```

Comment:
I tried to use `links.self` but it made the `GridLayout` all glitchy. Maybe I had something not quite right -- i'll try again.